demos/progress/js/progressDemo.js

```diff
@@ -84,7 +84,7 @@ var demo = demo || {};
         strings: {
             confirmStatus: "Confirm and submit your order.",
             orderSubmitted: "Order Submitted. Demo finished.",
-            percentCompleted: "%percent% Complete",
+            percentCompleted: "%percentComplete% Complete",
```

Comment:
`%percentComplete%` as a string template is also used in progress.js. So let's keep it this way.